### PR TITLE
AudioEngine:Fixed crash cause by multiple threads access into a shared unsynchronized data.

### PR DESCRIPTION
**Pull request by WenhaiLin**

### CODE REVIEW

**Comment by WenhaiLin** [comment]
[ci rebuild]

**Comment by pandamicro** [comment]
[ci rebuild]

**Comment by dabingnn** [comment]
@yangws Could you please review it?

File: cocos/audio/win32/AudioEngine-win32.cpp [hunk 1]
```diff
@@ -196,7 +196,12 @@ void AudioEngineImpl::_play2d(AudioCache *cache, int audioID)
         auto playerIt = _audioPlayers.find(audioID);
```
Comment (yangws) [inline cocos/audio/win32/AudioEngine-win32.cpp:196]: _audioPlayers is not thread safe yet.

**Pull request closed without merging**